pkgs/development/python-modules/manimpango/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonPackage
+, pkg-config
+, fetchPypi
+, cairo
+, pango
+, pygobject3
+, glib
+, libintl
+, harfbuzz
```

Comment:
don't much about nix packages but install `pango` should bring in all these dependencies right?
also, manimpango doesn't depend on `pygobject3`.